picking_good_slis.md

```diff
@@ -0,0 +1,92 @@
+# Picking Good SLIs
+
+## SRE's Philosophy on Monitoring
+
+A core goal of SRE is to ensure the availability of a service. "Availability"
+here means much more than simply "is my application running." Effectively
+monitoring a modern cloud application requires insight into the application's
+response time, error rate, traffic level, and more.
+
+Furthermore, the scope of monitoring should not be limited solely for use in
+deciding when to page an operations team. Metrics should be closely tied to
+business objectives, informing decisions on what features to prioritize and
+when to focus on adding new features versus improving reliability.
+
+To make this vision a reality, SRE teams rely on two key concepts:
+  * Service Level Indicators (SLIs) - A quantifiable metric that measures a
+    specific aspect of a service's availability.
+  * Service Level Objectives (SLOs) - A measurable, specific goal for a
+    given service's availability over time, based on one or more SLIs.
+
+As a starting point on towards realizing this goal, this document aims to
+provide guidance on selecting the right SLIs for your service.
+
+## Selecting SLIs
+
+For SRE teams looking to start monitoring their services, start by identifying
+SLIs to measure. Focus on measuring the basics to start, can refine them over
```

Comment:
I have added the point in a below comment with more details. I believe it is more useful to think through the SLOs that are most important for an end user of the service and then think through the SLIs.

As a separate effort, we can think through other SLIs which would be useful internally that would not have an associated SLO

Reply:
I copied this comment into the Google doc for further discussion.